src/status_im/common/standard_authentication/password_input/view.cljs

```diff
@@ -1,21 +1,22 @@
 (ns status-im.common.standard-authentication.password-input.view
   (:require
-    [clojure.string :as string]
-    [quo.core :as quo]
-    [quo.foundations.colors :as colors]
-    [quo.theme :as quo.theme]
-    [react-native.core :as rn]
-    [status-im.common.standard-authentication.forgot-password-doc.view :as forgot-password-doc]
-    [status-im.common.standard-authentication.password-input.style :as style]
-    [utils.debounce :as debounce]
-    [utils.i18n :as i18n]
-    [utils.re-frame :as rf]
-    [utils.security.core :as security]))
+   [clojure.string :as string]
+   [quo.core :as quo]
+   [quo.foundations.colors :as colors]
+   [quo.theme :as quo.theme]
+   [react-native.core :as rn]
+   [status-im.common.standard-authentication.forgot-password-doc.view :as forgot-password-doc]
+   [status-im.common.standard-authentication.password-input.style :as style]
+   [utils.debounce :as debounce]
+   [utils.i18n :as i18n]
+   [utils.re-frame :as rf]
+   [utils.security.core :as security]))
 
 (defn- get-error-message
   [error]
   (if (and (some? error)
            (or (= error "file is not a database")
+               (string/starts-with? error "failed to open database")
                (string/starts-with? error "failed to set ")
                (string/starts-with? error "Failed")))
```

Comment:
Maybe just use
```
(or (= error "file is not a database")
  (when error (string starts-with? (string/lower-case error) "failed")
```

Reply:
@Parveshdhull  am wondering why we have do we have a situation when the error would not start-with `file is not a database` otherwise why were other statements added